### PR TITLE
Learn: FAQ + Glossary

### PR DESCRIPTION
**Pull request by chasulin**
## TODO
Components
- [x] #1730 Add links to each section @pnavarrc 
- [x] #1730 Add the table of contents (mobile) @pnavarrc 
- [x] #1736 Breadcrumbs @pnavarrc 
- [x] #1741 Small styling fixes @pnavarrc
- [ ] #1734 Implement the sidebar table of contents (desktop) @pnavarrc 
- [ ] Add links to each question (we need to add ids on the CMS)
- [x] Make markdown styles more generic @chasulin 
- [x] Glossary @chasulin 
- [x] Landing page @chasulin 
- [ ] URLs in glossary definitions

[Bugs to fix](https://www.dropbox.com/scl/fi/npd55l5a2jcuio9fafhef/CAN-Learn.paper?dl=0&rlkey=fx0osez45obz0zywetu9lhpip)

Analytics and SEO
- [x] #1731 page title, URL and meta tags @pnavarrc 
- [ ] #1742 Tracking for expanding/collapsing questions  @pnavarrc 

Copy
- [x] FAQ intro
- [x] Landing page copy
- [x] Final meta description+title copy [from this doc](https://docs.google.com/document/d/10GKJXR4r3jmguoQRq98Ssw-aJFxIPmzgYBUrWvbxIwU/edit)
 
## Improvements
- [ ] Make the sidebar table of contents sticky and highlight items as the user scrolls down.
- [ ] Adjust the page layout

### CODE REVIEW

File: src/cms-content/learn/index.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import faq from './learn-faq.json';
+
+type Markdown = string;
+
+/*
+  For FAQ:
+*/
+export interface Question {
+  question: Markdown;
```
Comment (pnavarrc) [inline src/cms-content/learn/index.ts:9]: Maybe the question should be `string`?

File: src/screens/Learn/Faq/Faq.style.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export const PageHeader = styled.h1`
+  margin-bottom: 1.75rem;
+`;
+
+export const SectionHeader = styled.h2`
+  font-weight: 900;
+  font-size: 1.5rem;
+  line-height: 1.25;
+  margin: 1.25rem 0;
+`;
```
Comment (pnavarrc) [inline src/screens/Learn/Faq/Faq.style.tsx:19]: Maybe we should make these styles more generic, so we can reuse them in other pages on Learn (we can do that on a follow-up PR)

File: src/screens/Learn/Faq/Faq.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const faqHeader = faqContent.header;
+const faqSections = faqContent.sections;
+
+const Faq = () => {
+  return (
+    <Wrapper>
+      <PageHeader>{faqHeader}</PageHeader>
+      <p>*** insert table of contents ***</p>
+      {faqSections.map((section: any, i: number) => (
+        <FaqSection content={section} />
```
Comment (pnavarrc) [inline src/screens/Learn/Faq/Faq.tsx:15]: Consider fixing the type for the `section` and adding keys for each section (you can use the section ID, that's better than using the index)

File: src/screens/Learn/Faq/Faq.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import { Wrapper, PageHeader } from './Faq.style';
+import FaqSection from './FaqSection';
+import { faqContent } from 'cms-content/learn';
+
+const faqHeader = faqContent.header;
+const faqSections = faqContent.sections;
+
+const Faq = () => {
+  return (
+    <Wrapper>
```
Comment (pnavarrc) [inline src/screens/Learn/Faq/Faq.tsx:11]: We need to add the page title, canonical URL and description (see the LocationPage for an example)

```tsx
      <AppMetaTags
        canonicalUrl={canonicalUrl}
        pageTitle={pageTitle}
        pageDescription={pageDescription}
      />
```

File: src/screens/Learn/Faq/Faq.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Wrapper, PageHeader } from './Faq.style';
+import FaqSection from './FaqSection';
+import { faqContent } from 'cms-content/learn';
```
Comment (pnavarrc) [inline src/screens/Learn/Faq/Faq.tsx:4]: you can import the `Section` type here

File: src/screens/Learn/Faq/FaqSection.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+const FaqSection = (props: { content: Section }) => {
+  const { content } = props;
+  const { sectionTitle, questions } = content;
+
+  return (
+    <Fragment>
+      <SectionHeader>{sectionTitle}</SectionHeader>
+      {questions.map((item: Question, i: number) => (
+        <StyledAccordion summaryText={item.question} detailText={item.answer} />
```
Comment (pnavarrc) [inline src/screens/Learn/Faq/FaqSection.tsx:14]: Remember to add the key here (the ID is better)

**Review by pnavarrc — APPROVED** [review]
Looks good!

File: src/cms-content/learn/index.ts (hunk 1 above)
Comment (chasulin) [reply src/cms-content/learn/index.ts:9]: o yes youre right!

File: src/screens/Learn/Faq/Faq.tsx (hunk 4 above)
Comment (chasulin) [reply src/screens/Learn/Faq/Faq.tsx]: looks like you did this in a separate pr. gonna resolve!

File: src/screens/Learn/Faq/FaqSection.tsx (hunk 6 above)
Comment (chasulin) [reply src/screens/Learn/Faq/FaqSection.tsx:14]: there's no id on the question, using `accordion-question-${i}` instead

File: src/screens/Learn/Faq/Faq.style.tsx (hunk 2 above)
Comment (chasulin) [reply src/screens/Learn/Faq/Faq.style.tsx:19]: added a todo in the notes above

**Pull request merged**